docs/getting_started/installing_development_tools.md

```diff
@@ -0,0 +1,94 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+---
+
+(getting_started_installing_development_tools)=
+
+# Installing development tools
+
+To create and run workflows in Flyte, you must install Python, flytekit, flytectl, and Docker. Installing conda or another Python virtual environment manager is optional, but strongly recommended.
+
+## Install Python
+
+Python versions 3.8x - 3.10x are supported. [TK - is this accurate?]
```

Comment:
Just want to leave a note to not forget to update the question when you have an answer :)

Reply:
Python 3.x